san/transform.py

```diff
@@ -0,0 +1,30 @@
+import operator
+from san.query import convert_to_datetime_idx_df
+from functools import reduce
+
+QUERY_PATH_MAP = {
+    'eth_top_transactions': ['ethTopTransactions']
+}
+
+def path_to_data(idx, query, data):
+    return reduce(operator.getitem, ['query_'+str(idx),]+QUERY_PATH_MAP[query], data)
+    """
+    With this function we jump straight onto the key from the dataframe, that we want and start from there
+    """
```

Comment:
Is this comment for `path_to_data`? If so - it should be placed before the code, i.e. above `return`

Reply:
I am wondering why autopep8 doesn't catch those ? @spiderjako do you use it `autopep8  --in-place --aggressive *.py` before commiting ?

Reply:
autopep8 indeed formats some changes from this PR, though it does not change the comment. Formatting tools usually ignore comments

Reply:
I've used the autopep8, yes, but I guess, it didn't catch the comment, as Ivan suggested.

Reply:
And yes, the comment is for path_to_data